text/utilities/0000-syft-buildpack.md

```diff
@@ -0,0 +1,45 @@
+# Syft Buildpack
+
+## Summary
+
+Introduce a Syft buildpack. This buildpack will provide the `syft` binary should any buildpacks want to use it. The `syft` CLI is a tool and library for generating a Software Bill of Materials from container images and filesystems.
```

Comment:
I'd love to add a link to syft (https://github.com/anchore/syft) in this section.

Reply:
```suggestion
Introduce a Syft buildpack. This buildpack will provide the [`syft` binary](https://github.com/anchore/syft) should any buildpacks want to use it. The `syft` CLI is a tool and library for generating a Software Bill of Materials from container images and filesystems.
```

Reply:
How does that look?

Reply:
perfect!